### PR TITLE
using * in regex rather than {0,}

### PR DESCRIPTION
**Pull request by SilentSobs**
fix for #2917

### CODE REVIEW

**Comment by qqqttt123** [comment]
Could you correct your title and your description of pull request? You can refer to the format of other pull requests.

**Comment by zhoukangcn** [comment]
I found CI failed : https://github.com/datastrato/gravitino/actions/runs/8664502267

you can add a test case in `TestDorisUtils.java` 
```
    createTableSql = "CREATE DATABASE `it_db_342f1c85`\nPROPERTIES (\n\"property1\" = \"value1\",\n\"comment\" = \"comment\"\n)";
    result = DorisUtils.extractPropertiesFromSql(createTableSql);
    Assertions.assertEquals("value1", result.get("property1"));
    Assertions.assertEquals("comment", result.get("comment"));
```

File: catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public static Map<String, String> extractPropertiesFromSql(String createTableSql
     String[] lines = createTableSql.split("\n");
 
     boolean isProperties = false;
-    final String sProperties = "\"(.*)\"\\s{0,}=\\s{0,}\"(.*)\",?";
+    final String sProperties = "\"(.*)\"\\s*=\"(.*)\",?";
```
Comment (zhoukangcn) [inline catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java:35]: you’d better use `"\"(.*)\"\\s*=\\s*\"(.*)\",?"`

**Comment by zhoukangcn** [comment]
In order to accomplish milestone 0.5.0, I have made a fix in #3052

**Comment by shaofengshi** [comment]
Close this PR as there is no further action, thanks SilentSobs!

**Pull request closed without merging**